src/main/java/com/conveyal/gtfs/GTFSFeed.java

```diff
@@ -707,6 +709,26 @@ public double getTripSpeed (String trip_id, boolean straightLine) {
         return distance / time; // meters per second
     }
 
+    /** Returns sorted map of stop_ids mapped to tuple keys of trip_id, stop_sequence (for use in feed.stop_times) */
+    public SortedSet<Fun.Tuple2<String, Fun.Tuple2>> getStopTimesForStop (String stop_id) {
+        SortedSet<Fun.Tuple2<String, Fun.Tuple2>> index = this.stopStopTimeSet
+                .subSet(new Fun.Tuple2<>(stop_id, null), new Fun.Tuple2(stop_id, Fun.HI));
+
+        return index;
+    }
+
+    public ZoneId getTimeZoneForStop (String stop_id) {
```

Comment:
I think this function is named in confusing way; it implies that it's returning stop_timezone but it's returning agency_timezone.